oid_validation/pulp/oid_validation/oid_validation.py

```diff
@@ -29,7 +29,15 @@ def authenticate(environ, config=None):
     '''
     Framework hook method.
     '''
-    cert_pem = environ["mod_ssl.var_lookup"]("SSL_CLIENT_CERT")
+
+    # Attempt to retrieve the client certificate, and if it isn't available, reject.
+    cert_pem = ''
+    if 'SSL_CLIENT_CERT' in environ:
+        cert_pem = environ['SSL_CLIENT_CERT']
+    if not cert_pem and 'mod_ssl.var_lookup' in environ:
+        cert_pem = environ["mod_ssl.var_lookup"]("SSL_CLIENT_CERT")
+    if not cert_pem:
+        return False
```

Comment:
Can debug logs be added to this? It would be difficult to see what is happening without them.